scripts/quests/jeuno/LB03_Whence_Blows_the_wind.lua

```diff
@@ -0,0 +1,134 @@
+local quest = Quest:new(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.WHENCE_BLOWS_THE_WIND)
+
+quest.reward =
+{
+    fame = 50,
+    fameArea = JEUNO,
+    title = xi.title.SKY_BREAKER,
+}
+
+quest.sections =
+{
+    -- Section: Quest available.
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getMainLvl() >= 56 and
+                player:getLevelCap() == 60 and
+                xi.settings.MAX_LEVEL >= 65
+        end,
+
+        [xi.zone.RULUDE_GARDENS] =
+        {
+            ['Maat'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(85)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [85] = function(player, csid, option, npc)
+                    if option == 1 then -- Accept quest option.
+                        quest:begin(player)
+                    end
+                end,
+            },
+        },
+    },
+
+    -- Section: Quest accepted.
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.RULUDE_GARDENS] =
+        {
+            ['Maat'] =
+            {
+                onTrigger = function(player, npc)
+                    if
+                        player:hasKeyItem(xi.ki.ORCISH_CREST) and
+                        player:hasKeyItem(xi.ki.QUADAV_CREST) and
+                        player:hasKeyItem(xi.ki.YAGUDO_CREST)
+                    then
+                        return quest:progressEvent(87)
+                    else
+                        return quest:event(86)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [87] = function(player, csid, option, npc)
+                    if quest:complete(player) then
+                        player:delKeyItem(xi.ki.ORCISH_CREST)
+                        player:delKeyItem(xi.ki.QUADAV_CREST)
+                        player:delKeyItem(xi.ki.YAGUDO_CREST)
+                        player:setLevelCap(65)
+                        player:messageSpecial(ruludeID.text.YOUR_LEVEL_LIMIT_IS_NOW_65)
+                    end
+                end,
+            },
+        },
+
+        [xi.zone.CASTLE_OZTROJA] =
+        {
+            ['qm2'] =
+            {
+                onTrigger = function(player, npc)
+                    if not player:hasKeyItem(xi.ki.YAGUDO_CREST) then
+                        return npcUtil.giveKeyItem(player, xi.ki.YAGUDO_CREST)
+                    end
+                end,
+            },
+        },
+
+        [xi.zone.MONASTIC_CAVERN] =
+        {
+            ['qm1'] =
+            {
+                onTrigger = function(player, npc)
+                    if not player:hasKeyItem(xi.ki.ORCISH_CREST) then
+                        return npcUtil.giveKeyItem(player, xi.ki.ORCISH_CREST)
```

Comment:
One additional note, for these onTrigger returns, it needs to be from the mission class to not trigger the default action if there is one.  Should be returning `mission:keyItem(xi.ki.ORCISH_CREST)` and similar for the others